modules/react/dialog/stories/stories_FallbackTesting.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {withSnapshotsEnabled, PopperController, customViewport} from '../../../../utils/storybook';
+import {Placement} from '@workday/canvas-kit-react/popup';
+import {Tooltip} from '@workday/canvas-kit-react/tooltip';
+import {Dialog} from '@workday/canvas-kit-react/dialog';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {Flex} from '@workday/canvas-kit-react/layout';
+
+export default {
+  title: 'Testing/Popups/Dialog',
+  component: Dialog,
+  parameters: {
+    viewport: {
+      viewports: customViewport,
+      defaultViewport: 'landscape',
+    },
+  },
+};
+
+export const DialogWithFallbackPlacements = withSnapshotsEnabled(() => {
```

Comment:
We don't need Chromatic snapshots for this if the dialog isn't visible. We only have a limited number of snapshots available per month before we get charged more money.

```suggestion
export const DialogWithFallbackPlacements = () => {
```